pkg/capabilities/consensus/ocr3_capability.go

```diff
@@ -0,0 +1,136 @@
+package consensus
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+)
+
+var info = capabilities.MustNewCapabilityInfo(
+	"ocr3",
+	capabilities.CapabilityTypeConsensus,
+	"OCR3 consensus exposed as a capability.",
+	"v1.0.0",
+)
+
+type ocr3Capability struct {
+	capabilities.CapabilityInfo
+	store     *store
+	evictedCh chan []*request
+}
+
+func newOCR3Capability(ctx context.Context, s *store) *ocr3Capability {
+	o := &ocr3Capability{
+		CapabilityInfo: info,
+		store:          s,
+		evictedCh:      make(chan []*request),
+	}
+	go o.evictionLoop(ctx)
```

Comment:
nit: Starting goroutines in constructors always worries me. Don't we prefer to follow the ServiceCtx interface and use Start/Stop methods?